### PR TITLE
Transfer h

### PR DESCRIPTION
**Pull request by HUANG0421**
I have checked what Oliver has already done before. Now here is the missing part from h files.

> Pull-requests should be ready to merge, and quick to review. Please help your reviewer by using this form. If you require a specific reviewer you can select them on the right. 

### Please indicate relevant issues
See # .
Close # .

### Checklist
-   [x] The title line explains what these changes do
-   [x] All my changes are marked in the `last_modfied` column
-   [x] All formatting changes are intentional (dates, coordinates)
-   [x] My changes do not add empty rows
-   [x] I checked the modified table on GitHub (click `view`)

### CODE REVIEW

File: data/Act.csv [hunk 1]
```diff
@@ -497,3 +497,18 @@ A00721,P0120,AT05,PS09999,,196x..197x,196x..197x,Zhang mentions it as item liste
 A00722,P0020,AT01,PS00468,,1972..197x,1972..197x,unknown,SS00006,137,2,,,2018-07-04,MG,2018-07-04,MG
 A00723,P0173,AT03,PS00468,P0020,1972..197x,1972..197x,unknown,SS00006,137,2,,,2018-07-04,MG,2018-07-04,MG
 A00724,P0173,AT19,PS00468,P0020,1972..197x,1972..197x,Liu and Gao discussed Russian literature frequently,SS00006,137,2,,,2018-07-04,MG,2018-07-04,MG
+,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,
+A00517,P0011,AT05,AW09999,,1968..,1968..,The author was astonished when she saw the works of these renowned painters in the houses of her friends whose parents are painters as well.,SS00155,56,2,,,2018-06-21,WH,2018-06-21,WH
+A00518,P0031,AT05,PS00407,,1969..,1969..,"The author met Zhu Shixian in the prison, Zhu told him about some good classical texts from a high school textbook.",SS00109,ONLINE SOURCE,2,,,2018-06-21,WH,2018-06-21,WH
+A00518,P0032,AT05,PS00407,,1969..,1969..,"He had read some classical texts from a high school textbook, he told Huang Guangzu in the prison that he thought these articles are very good.",SS00109,ONLINE SOURCE,2,,,2018-06-21,WH,2018-06-21,WH
```
Comment (duncdrum) [inline data/Act.csv]: There are two acts `A00518` each act should have its own id.
Do Huang Guangzu and Zhu Shixian have person ids. 
Each descriptions should be two sentences.

File: data/Act.csv [hunk 2]
```diff
@@ -497,3 +497,18 @@ A00721,P0120,AT05,PS09999,,196x..197x,196x..197x,Zhang mentions it as item liste
...
+A00519,P0055,AT13,PS00408,,195x,195x,He accomplished the book series.,SS00131,ONLINE SOURCE,2,,,2018-06-21,WH,2018-06-21,WH
+A00520,P0031,AT01,PS00409,,1969..,1969..,Huang got in to the Sichuan Second Prison in 1969 Summer due to his diary. He was sent to labor reform and he met Zhu Shixian there. The text is about their lives in the prison. They recall and read many literature in the prison. Zhu suicided in the prison.,SS00109,ONLINE SOURCE,2,,,2018-07-09,WH,2018-07-09,WH
```
Comment (duncdrum) [inline data/Act.csv]: Sichuan Second Prison should either be a place or a location

File: data/Act.csv [hunk 3]
```diff
@@ -497,3 +497,18 @@ A00721,P0120,AT05,PS09999,,196x..197x,196x..197x,Zhang mentions it as item liste
...
+A00521,P0032,AT01,PS00409,,19xx,19xx,"Zhu read the text before he was send to the prison, he also has it in the prison and read it with Huang Guangzu.",SS00109,ONLINE SOURCE,2,,,2018-07-09,WH,2018-07-09,WH
```
Comment (duncdrum) [inline data/Act.csv]: this reads awkward, try reformulating without commas 😉

File: data/Act.csv [hunk 4]
```diff
@@ -497,3 +497,18 @@ A00721,P0120,AT05,PS09999,,196x..197x,196x..197x,Zhang mentions it as item liste
...
+A00522,P0055,AT13,PS00410,,195x,195x,He accomplished the book series.,SS00131,ONLINE SOURCE,2,,,2018-07-09,WH,2018-07-09,WH
+A00523,P0055,AT13,PS00411,,195x,195x,He accomplished the book series.,SS00131,ONLINE SOURCE,2,,,2018-07-10,WH,2018-07-10,WH
+A00524,P0055,AT13,PS00412,,195x,195x,He accomplished the book series.,SS00131,ONLINE SOURCE,2,,,2018-07-10,WH,2018-07-10,WH
+A00525,P0046,AT01,PS00413,,1971,1971,"In the study small class, study members started to read other random books after reading the Little Red Book.",SS00120,ONLINE SOURCE,2,,,2018-07-10,WH,2018-07-10,WH
```
Comment (duncdrum) [inline data/Act.csv]: rephrase

File: data/Act.csv [hunk 5]
```diff
@@ -497,3 +497,18 @@ A00721,P0120,AT05,PS09999,,196x..197x,196x..197x,Zhang mentions it as item liste
...
+A00526,P0109,AT05,PS00414,,1966..1970~,1966..1970~,Shu Ting relates that she read the Novel as she was hit by a bullet fired by red guards invading the physics building,SS00014,300,2,,,2018-07-12,WH,2018-07-12,WH
+A00527,P0106,AT05,PS00414,,196x,196x,"At the time he read widely but only, and remembers these when he later saw these titles again",SS00005,565,2,,,2018-07-12,WH,2018-07-12,WH
```
Comment (duncdrum) [inline data/Act.csv]: rephrase meaning unclear

File: data/Person.csv [hunk 6]
```diff
@@ -819,3 +819,13 @@ P0523,Feng,Zhi,en,male,NULL,NULL,NULL,NULL,1905,1993,Feng Chengzhi,en,PL0183,,,h
 P0523,冯,至,zh,male,NULL,NULL,NULL,NULL,1905,1993,冯承植,zh,PL0183,,,https://zh.wikipedia.org/wiki/%E5%86%AF%E8%87%B3,,,,,,2018-05-03,WH,2018-05-03,WH
 P0524,Fu,Chongju,en,male,NULL,NULL,NULL,NULL,1875,1917,,,PL0352,,,https://book.douban.com/subject/1733761/,,,,,,2018-05-03,WH,2018-05-03,WH
 P0524,傅,崇矩,zh,male,NULL,NULL,NULL,NULL,1875,1917,,,PL0352,,,https://book.douban.com/subject/1733761/,,,,,,2018-05-03,WH,2018-05-03,WH
+P0525,Rembrandt,Harmenszoon van Rijn,en,male,NULL,NULL,NULL,NULL,1606,1669,NULL,NULL,PL0353,,,https://zh.wikipedia.org/wiki/%E4%BC%A6%E5%8B%83%E6%9C%97,,,,,,2018-06-21,WH,2018-06-21,WH
```
Comment (duncdrum) [inline data/Person.csv]: is this consistent with other entries, as being `en` or should this be dutch?

**Comment by duncdrum** [comment]
@HUANG0421 a live-sign about this pull request please

File: data/Person.csv (hunk 6 above)
Comment (duncdrum) [reply data/Person.csv]: actually it is consistent so no reason to change

**Comment by HUANG0421** [comment]
Actually I worked with this PR yesterday. But the question is that there has a big change in Act File and I really need a bit more time for it. So Sorry and I will finish it in the next 2 days. :)

**Pull request merged**